hieradata/nodes/fireball.yaml

```diff
@@ -1,6 +1,6 @@
 classes:
     - ocf::packages::docker
     - ocf_ocfweb::dev_config
-    - ocf_ssl
+    - ocf_ssl::default_bundle
```

Comment:
I think this is a good example of why managing classes via hiera is much nicer than LDAP. Previously I would have had to try to coordinate this change when pushing this to master, making sure people didn't revert it in their branches if they changed the environment without pulling master, etc.